### DISCUSSION TITLE
What is the difference between web3j and web3j-android?

### DISCUSSION THREAD

**Discussion by soharang** [web3j/web3j#1728] [Q&A]
Hello, 

Is it OK that I can apply v4.9.2 on my android application?

When I check releases of web3j on github, the latest version is v4.9.2 for web3j and v4.8.8 for web3j-android.
For now, I am using v4.8.7-android on my android application and I would like to apply the latest version. 

It seems that there is a difference between normal version and android version, for example, android version excludes some APIs not available on Android.

Thank you.

**Comment by muratogat** [comment]
I really would like to get an answer for that, since the android branch always lags behind. 
However, there is that:
https://github.com/web3j/web3j/compare/v4.8.8...v4.8.8-android -> 4.8.8 and 4.8.8-android are identical
https://github.com/web3j/web3j/compare/v4.8.7...v4.8.7-android -> so are 4.8.7 and 4.8.7-android
https://github.com/web3j/web3j/compare/v4.6.0...v4.6.0-android -> 4.6.0 and 4.6.0-android have differences, mainly about Java8 and streams it seems
